src/__tests__/definitions/valid-parallel-nested.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Comment": "Contains a nested parallel state",
+  "StartAt": "Parallel",
+  "States": {
+    "Parallel": {
+      "Type": "Parallel",
+      "Next": "Final State",
```

Comment:
added for better coverage of the JSON path expressions to evaluate nested states

the bug fix for #84 didn't include a test for nested parallel